tests/auth0/handlers/hooks.tests.js

```diff
@@ -124,6 +124,12 @@ describe('#hooks handler', () => {
           code: 'code',
           active: false,
           triggerId: 'credentials-exchange'
+        },
+        {
+          name: 'Hook-6',
+          code: 'code',
+          active: false,
+          triggerId: 'post-change-password'
```

Comment:
I noticed there is no assertion here, have you tried to fail this test?

Reply:
Yikes, it does not fail somehow if I remove the trigger from the allowed list. Will investigate.

Reply:
The unit test here only exercise the handler-specific `validate()` function. Since the enum for the triggerId is defined in the schema, the validation is only exercise in the validator test cases, which is also missing for hooks.
I've added hooks validator tests in 5971ea0.

Reply:
Thank you, those look good. I see that distinction now.